### DISCUSSION TITLE
How to change mouse cursor style?

### DISCUSSION THREAD

**Discussion by Kritiqual** [wezterm/wezterm#2404] [Q&A] [answered]
When I hover the tab bar, the mouse cursor is an arrow in a weird angle. So, how can I change the mouse cursor?

OS: Archlinux
Wezterm version: 20220807-113146-c2fee766

**Comment by wez** [comment]
Sounds like an xcursor theme related issue. Please take a look through [these issues](https://github.com/wez/wezterm/search?q=xcursor&type=issues) to learn more about how cursors are resolved, and if that doesn't help, please file an issue with more information about your system.

**Comment by Kritiqual** [comment, marked as answer]
Based on this idea of [issue 524 (comment)](https://github.com/wez/wezterm/issues/524#issuecomment-1008363963), I added `Xcursor.theme: Sweet-cursors` in `~/.Xresources` and after a reboot, wezterm now use the correct theme
